docs/application/tizen-studio/native-tools/ui-builder/component-attributes.md

```diff
@@ -514,7 +514,7 @@ The following attributes are used for the entry component.
 
 ### Flipselector (![Flipselector](./media/component_attributes_flipselector_icon.png))
 
-A [flipselector](../../native/ui/efl/component-flipselector-m.md) component shows a set of text items one at a time. The user can flip the selector up or down to change the text on it. It can contain a small number of items. **(Since 2.4, this component is supported in mobile applications only.)**
+A [flipselector](../../../native/ui/efl/mobile/component-flipselector.md) component shows a set of text items one at a time. The user can flip the selector up or down to change the text on it. It can contain a small number of items. **(Since 2.4, this component is supported in mobile applications only.)**
```

Comment:
It looks `../../../native/guides/ui/efl/mobile/component-flipselector.md` is correct

Reply:
done